plugins/jira-dashboard/src/components/JiraDashboardContent/JiraDashboardContent.tsx

```diff
@@ -77,6 +82,11 @@ export const JiraDashboardContent = (props?: {
                   tableContent={value}
                   showFilters={props?.showFilters}
                   project={jiraResponse.project}
+                  tableOptions={{
```

Comment:
The props need to be passed on to the "JiraTable". Remove all new options and add this?
```
tableOptions={...tableOptions}
```